daffodil-japi/src/main/scala/org/apache/daffodil/japi/schema/RuntimeSchemaWalker.scala

```diff
@@ -0,0 +1,135 @@
+package org.apache.daffodil.japi.schema
```

Comment:
I wonder if this wants to be in some experimental namespace? I consider japi and sapi as being very strict about backwards compatibility. It might be nice to put this in a namespace that makes it clear this maybe not be final and is subject to change--I don't want to tie us down to an API that maybe isn't completely fleshed out.

Reply:
How about japix which we document as japi-experimental?